frontend/src/components/MeetingRoom/DeviceSettingsMenu/DeviceSettingsMenu.spec.tsx

```diff
@@ -1,6 +1,6 @@
 import { act, queryByText, render, screen, waitFor } from '@testing-library/react';
 import { describe, beforeEach, it, Mock, vi, expect, afterAll } from 'vitest';
-import { MutableRefObject } from 'react';
+import { RefObject } from 'react';
```

Comment:
For React 19, all `RefObject`s are implicitly mutable now?

Reply:
that is correct, mutablerefobject has been deprecated and it's all on refobject now